roles/network_plugin/multus/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Multus | Delete existing configuration
+  file:
+    state: absent
+    path: "{{ multus_conf_dir }}/{{ multus_conf_filename }}"
+
+- name: Multus | Read CNI configuration of master plugin
+  shell: |
```

Comment:
Try to use native modules when possible, instead of command/shell: https://docs.ansible.com/ansible/2.5/modules/slurp_module.html